Resources/Audio/Expedition/attributions.yml

```diff
@@ -0,0 +1,9 @@
+- files: ["tension_session.ogg"]
+  license: "CC-BY-3.0"
+  copyright: "Created by qwertyquerty"
+  source: "https://www.youtube.com/@qwertyquerty"
+
+- files: ["deadline.ogg"]
+  license: "CC-BY-3.0"
```

Comment:
I asked about this because it's 4.0 on discord but DZ said same license as all other songs.